src/libsync/syncfilestatustracker.cpp

```diff
@@ -68,6 +68,12 @@ static inline bool showWarningInSocketApi(const SyncFileItem& item)
         || status == SyncFileItem::Restoration;
 }
 
+static inline bool showSyncInSocketApi( const SyncFileItem& item)
+{
+    const auto inst = item._instruction;
```

Comment:
You do :-)

it is explained in the commit message https://github.com/owncloud/client/commit/361ebf5464077ce08049ea9e1f834e6fe56cade9